daisy/compute/compute.go

```diff
@@ -261,7 +275,15 @@ func NewClient(ctx context.Context, opts ...option.ClientOption) (Client, error)
 	if ep != "" {
 		rawBetaService.BasePath = ep
 	}
-	c := &client{hc: hc, raw: rawService, rawBeta: rawBetaService}
+	rawAlphaService, err := computeAlpha.New(hc)
+	if err != nil {
+		return nil, fmt.Errorf("alpha compute client: %v", err)
+	}
```

Comment:
I recall @adjackura having some thoughts regarding Alpha APIs.

Reply:
Alpha API won't work if they are not enabled. In our case, it's expected. It happens to only shadow logic and has no side effect.